edx_proctoring/backends/rest.py

```diff
@@ -8,6 +8,9 @@
 import uuid
 import jwt
 
+# this is a known PyLint false positive (https://github.com/PyCQA/pylint/issues/1640)
+# that is fixed in PyLint v1.8
```

Comment:
Would the tools team be the right group to ask about how difficult it would be to upgrade pylint?

Reply:
I can bug them. I asked Cale about it and he said it would be a pain, but I'll double check.

Reply:
(for posterity) I'm satisfied with Cale's vibe